load-issues.js

```diff
@@ -2,20 +2,58 @@ var Issue = require('./issue.js');
 var config = require('./config.js');
 var debug = require('debug')('backlog:load-issues');
 var github = require('./github.js');
+var paraflow = require('paraflow');
 
-module.exports = function getIssues() {
+// Max number of github requests in parallel
+const PARALELL_LIMIT = 5;
+
+module.exports = () => {
    return github.getOpenBacklogIssues()
    .then(createIssuesFromGithubResponses)
+   .then(getPreviousLabelsForIssues)
 }
 
 function createIssuesFromGithubResponses(githubIssues) {
-   return githubIssues.map(createIssueFromGithubResponse);
+   return githubIssues.map(createIssueFromGithubResponse)
```

Comment:
Why did you remove the semicolons in this file?

Reply:
Cause I'm trying to get used to not seeing them. They aren't needed in 99% of cases and I'm trying to let go of my FUD about seeing them :-). No real functional reason.